entity-service-api/src/main/proto/org/hypertrace/entity/type/service/v2/entity_type_service.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+package org.hypertrace.entity.type.service.v2;
+
+message EntityType {
+  string name = 1;
```

Comment:
what's the difference between the name and scope? I guess it doesn't hurt to separate them, but we'd expect them to be the same right?

Reply:
Yes, same for the cases we have so far. When we bring in infra entities, it might change though hence keeping it.